app/models/users/notification_setting.rb

```diff
@@ -0,0 +1,16 @@
+module Users
+  class NotificationSetting < ApplicationRecord
+    self.table_name_prefix = "users_"
+
+    validates :user_id, presence: true
+    validates :email_digest_periodic, inclusion: { in: [true, false] }
+
+    alias_attribute :subscribed_to_welcome_notifications?, :welcome_notifications
+    alias_attribute :subscribed_to_mod_roundrobin_notifications?, :mod_roundrobin_notifications
+    alias_attribute :subscribed_to_email_follower_notifications?, :email_follower_notifications
+
+    # NOTE: @ridhwana Need to account for
```

Comment:
This validation was more complicated to move across, so I left a note for myself